src/components/Sidebar/ElementProfileOptions.vue

```diff
@@ -1,9 +1,23 @@
 <template>
   <div class="c-element-profile-options">
-    <div class="btn-group w-100" role="group" aria-label="Navigation">
-      <router-link :to="'/element/' + previousElementId" tag="button" type="button" class="btn btn-primary w-100" :disabled="previousElementId === 0">{{ $t("sidebar.elementProfile.previous") }}</router-link>
-      <router-link :to="'/element/' + nextElementId" tag="button" type="button" class="btn btn-primary w-100" :disabled="nextElementId === numberOfElements + 1">{{ $t("sidebar.elementProfile.next") }}</router-link>
```

Comment:
If the words "next" and "previous" are no longer useful, you should remove the traductions from the i18n's files.